crd-spec.yaml

```diff
@@ -1,6 +1,5 @@
 apiVersion: cactuar.rs/v1
-# I'd like to change this name to ServiceAlerts. Thoughts?
```

Comment:
Thoughts on deleting this file until we're done and stabilised? The test we have is currently canon

Reply:
Yeah let's go ahead and nuke that file. 👍